### PR TITLE
upgrade jquery to v3 and implement photoswipe

### LINKED ISSUE #2278: Images - Enable zoom on mobile
It would be quite great if, on mobile, we could zoom on images once opened.

### PR DESCRIPTION
**Pull request by karlprieb**
@RocketChat/core 

Closes #2278

### CODE REVIEW

File: packages/rocketchat-oembed/client/oembedImageWidget.html [hunk 1]
```diff
@@ -5,13 +5,11 @@
...
 			{{#if loadImage}}
-				<div>
-					<a href="{{url}}" class="swipebox" target="_blank">
-						<div class="inline-image" style="background-image: url('{{url}}');">
-							<img src="{{url}}" height="200">
-						</div>
-					</a>
-				</div>
+				<figure>
+					<div class="inline-image" style="background-image: url('{{url}}');">
```
Comment (engelgabriel) [inline packages/rocketchat-oembed/client/oembedImageWidget.html]: Do we still need this DIV with the background-image?

File: packages/rocketchat-ui-admin/admin/admin.coffee [hunk 2]
```diff
@@ -472,9 +472,7 @@ Template.admin.onRendered ->
 		SideNav.openFlex()
 
 	Tracker.autorun ->
-		FlowRouter.watchPathChange()
```
Comment (engelgabriel) [inline packages/rocketchat-ui-admin/admin/admin.coffee]: Why was this removed?

File: public/images/default-skin.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...5v2h-3v3h-2v-5zM31 15v5h-2v-3h-3v-2h5zM31 29h-5v-2h3v-3h2v5z" id="Shape"/></g><g fill="#fff"><path d="M62 24v5h-2v-3h-3v-2h5zM62 20h-5v-2h3v-3h2v5zM70 20v-5h2v3h3v2h-5zM70 24h5v2h-3v3h-2v-5z"/></g><path d="M20.586 66l-5.656-5.656 1.414-1.414L22 64.586l5.656-5.656 1.414 1.414L23.414 66l5.656 5.656-1.414 1.414L22 67.414l-5.656 5.656-1.414-1.414L20.586 66z" fill="#fff"/><path d="M111.785 65.03L110 63.5l3-3.5h-10v-2h10l-3-3.5 1.785-1.468L117 59l-5.215 6.03z" fill="#fff"/><path d="M152.215 65.03L154 63.5l-3-3.5h10v-2h-10l3-3.5-1.785-1.468L147 59l5.215 6.03z" fill="#fff"/><g><path id="Rectangle-11" fill="#fff" d="M160.957 28.543l-3.25-3.25-1.413 1.414 3.25 3.25z"/><path d="M152.5 27c3.038 0 5.5-2.462 5.5-5.5s-2.462-5.5-5.5-5.5-5.5 2.462-5.5 5.5 2.462 5.5 5.5 5.5z" id="Oval-1" stroke="#fff" stroke-width="1.5"/><path fill="#fff" d="M150 21h5v1h-5z"/></g><g><path d="M116.957 28.543l-1.414 1.414-3.25-3.25 1.414-1.414 3.25 3.25z" fill="#fff"/><path d="M108.5 27c3.038 0 5.5-2.462 5.5-5.5s-2.462-5.5-5.5-5.5-5.5 2.462-5.5 5.5 2.462 5.5 5.5 5.5z" stroke="#fff" stroke-width="1.5"/><path fill="#fff" d="M106 21h5v1h-5z"/><path fill="#fff" d="M109.043 19.008l-.085 5-1-.017.085-5z"/></g></g></g></svg>
```
Comment (engelgabriel) [inline public/images/default-skin.svg]: Have we edited it from the original? Why did we not just add this as an asset directly from NPM?

File: packages/rocketchat-ui-admin/admin/admin.coffee (hunk 2 above)
Comment (karlprieb) [inline packages/rocketchat-ui-admin/admin/admin.coffee]: `FlowRouter.getParam` generate the reactivity needed.
